src/Maintenance/Maintenance/ChangeLog.md

```diff
@@ -20,6 +20,8 @@
 
 ## Upcoming Release
 * Fixed bug where rebootSettings property wasn't updating.
+* Fixed bug where Get-MaintenanceConfig was returning a List.
```

Comment:
```suggestion
* Fixed bug where `Get-AzMaintenanceConfiguration` was returning a List. [#25781]
```
- ensure official name of cmdlet used in our documentation.
- no need to explain bug in change log but add the link of github issue.

Reply:
Added this change :)